docs/introduction.md

```diff
@@ -193,3 +193,55 @@ _(see: [schema.graphql](https://github.com/ledger-subquery/blob/main/schema.grap
 
 ![entity database relationship diagram](./assets/entities_db.svg)
 ![entity api relationship diagram](./assets/entities_api.svg)
+
+## Versioning
+The versions of both the GraphQL API and the Indexer itself can be retrieved simply using the following query on the GraphQL playground.
+
+##### Example:
+
+```graphql
+query ReleaseVersionTest {
+  _metadata {
+    queryNodeVersion
+    indexerNodeVersion
+  }
+}
+```
+
+Each of these version numbers are stored as the value to the key `"version"` within their relevant module `package.json` file. These files can be found in the `docker/node-cosmos/node_modules` and `subql/packages/query/` directories for the Indexer and GraphQL versions, respectively.
```

Comment:
```suggestion
Each of these version numbers are stored as the value to the key `"version"` within their relevant module `package.json` file. These files can be found in the `docker/node-cosmos/` and `subql/packages/query/` directories for the Indexer and GraphQL versions, respectively.
```

Reply:
In my local repo the `package.json` path is "docker/node-cosmos/package.json" - am I meant to be referencing the `@suql` directory in docker/node-cosmos/node-modules`?